### ISSUE TITLE
Needs a dedicated man page

### ISSUE THREAD

**Issue by FichteFoll** [kennethreitz/pipenv#1100]
So, pipenv ships with a man page that can be opened with `pipenv --man`, but that man page has a few issues regarding usability, because it is just built from the sphinx docs.

1. I don't need any information about how to install pipenv. I already have it. (I don't need user testimonials either.)
2. It doesn't list the general cli usage at the top, which is one of the most useful sections.
3. The docs have a couple of links, which don't convert to a man page.
4. Example files are good, but more useful at the bottom
5. More, that I don't care to list right now.

Basically, it's not really a man page. It's a user guide in form of a man page.
That said, it's still good that there is *something* you can access (without an internet connection), but in the current state it's pretty lackluster. Pipenv needs a dedicated man page imho.

**Comment by erinxocon** [comment]
@FichteFoll I would agree.  It is an issue we should tackle, but not one that is a priority right now.  Anyone want to submit a PR? :)

**Comment by kennethreitz** [comment]
that's not how pip works

**Comment by FichteFoll** [comment]
What does this have to do with pip?

**Comment by kennethreitz** [comment]
we can't ship a proper man page that works with man, i've looked into it
heavily.

we can improve the "man" functionality, but i'd rather just remove it.

**Comment by techalchemy** [comment]
wouldn't we have to do distro packaging for actual man pages? I've made them before but it was a pain

**Comment by FichteFoll** [comment]
I was mainly referring to the `pipenv --man` functionality here, not requesting that the man page is added to the system's database. That would indeed be a task for distro packaging (and certainly not hard for Arch, but I don't know about others).

**Comment by erinxocon** [comment]
@kennethreitz I didn't mean we should create a man page, I remember us trying to get it to install a man page and we couldn't.  I just thought about changing the `pipenv --man` link to look more like a traditional man page instead of the sphinx docs.  As I said though not a priority right now :)

**Comment by kennethreitz** [comment]
i dont’ think the current —man is harmful

**Comment by funkybob** [comment]
I don't think the issue is about having --man... I think it's more the formatting, content, and sequence of information.

I read this more as a request for improvement of the existing --man output.

**Comment by uranusjr** [comment]
We don’t know how to answer the request. If you do, do it for us!

**Comment by FichteFoll** [comment]
I picked up some troff/groff syntax and proper man page structure over the weekend. Would you prefer writing in groff directly or would you rather have something that compiles to it, like a pod for example (other suggestions welcome).

**Comment by FichteFoll** [comment]
Just a note that I started working on this, but will probably take another 2 weeks to finish because of a vacation.

**Comment by FichteFoll** [comment]
So, initially I thought of having everything in a single man page, but eventually I realized that the number of sub commands and their options would become quite cumbersome to maintain and read in a single file. Thus, I propose to have a separate page for each command and one that mentions them all, along with some general usage considerations, examples, file explanations and environment variables. The only "problem" is that this doesn't work well with the `--man` flag currently. I suppose an argument could be added for that to open a subcommand's page, though.

Would that be okay?

**Comment by techalchemy** [comment]
I don’t think there’s anything wrong with big man pages, it’s pretty common. And it’s not that common to split things up. As a user of man pages I’d probably find that annoying
